### ISSUE TITLE
hung run script - yoshi build --analyze

### ISSUE THREAD

**Issue by tompere** [wix/yoshi#280]
# 🐛 Bug Report

### 😯 Current Behavior

`yoshi build --analyze` is hung
...

### 🤔 Expected Behavior

`yoshi build --analyze` should work 
...

### 🔦 Context 

...

### 💁 Possible Solution 

...

### 💻 Code Sample

https://github.com/wix-private/crm/tree/master/crm-wixcode-js-commons
...

### 🌍 Your Environment

```

  System:
    OS: macOS Sierra 10.12.5
    CPU: x64 Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz
  Binaries:
    Node: 8.9.1 - ~/.nvm/versions/node/v8.9.1/bin/node
    Yarn: 1.3.2 - ~/.yarn/bin/yarn
    npm: 5.6.0 - ~/.nvm/versions/node/v8.9.1/bin/npm
  Browsers:
    Chrome: 66.0.3359.139
    Safari: 10.1.1
  npmPackages:
    yoshi: ^2.1.2 => 2.3.0

 Yoshi config
{
  "hmr": "auto"
}
```

**Comment by ronami** [comment]
Should be solved by https://github.com/wix/haste/pull/190.

**Comment by ranyitz** [comment]
Fixed on version [v2.7.0](https://github.com/wix/yoshi/blob/master/CHANGELOG.md#270-may-31-2018)
